app/controllers/admin/community_customizations_controller.rb

```diff
@@ -5,15 +5,23 @@ def edit_details
     @selected_left_navi_link = "tribe_details"
     # @community_customization is fetched in application_controller
     @community_customizations ||= find_or_initialize_customizations(@current_community.locales)
-    all_locales = MarketplaceService::API::Marketplaces.all_locales
+    all_locales = MarketplaceService::API::Marketplaces.all_locales.map{|l| l[:locale_key]}
     enabled_locale_keys = available_locales.map(&:second)
 
+    custom_translation_in_use = !all_locales.include?(@current_community.locales.first)
```

Comment:
`custom` is not very good name here, since it strongly refers to per marketplace custom languages, which will be removed. I vote for `unofficial` or something like that.

(not sure if that needs to be changed right now)